### PR TITLE
Fix popups

### PR DESCRIPTION
**Pull request by tim-band**
On the text details view, for example search ugu -> watering place -> KUB 06 -> hover over the blue "1", "2" or "3" in the middle. On production clicking these brings up a blank popup. With this change hovering gives details of the original text, like in the old oracc.

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -29,7 +29,9 @@
               "src/styles.scss",
               "./node_modules/@fortawesome/fontawesome-free/css/all.min.css"
             ],
-            "scripts": []
+            "scripts": [
+              "src/scripts/p3.js"
+            ]
```
Comment (jhughes982) [inline angular.json:34]: out of curiosity, what is p3.js? Does the code come from a library?

**Review by jhughes982 — APPROVED** [review]
Looks good!

**Comment by jhughes982** [comment]
Tested locally and the functionality all works ok. Once this is merged in you could do a fresh deploy to prod.

File: angular.json (hunk 1 above)
Comment (tim-band) [reply angular.json:34]: Originally I stole it from Steve's code, but now it looks nothing like it!

**Pull request merged**